feat: Supplier Sourced Item in Subcontracting (#168)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/manufacturing/subcontracting.md

<img class="screenshot" alt="Sub-Contracting" src="{{docs_base_url}}/assets/img/manufacturing/subcontract5.png">

### 1.5 Supplier Sourced Raw Material

> If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.

Reviewer: ```suggestion
While creating a BOM for subcontracting, there might be raw material which Supplier will have to procure themselves like nuts and bolts. While creating a Stock Entry for Transfer from Purchase Order, these items can be excluded one by one but it is impossible to do so if you have more than 100 items. 

If some raw material is sourced by the Supplier directly, such raw material has to be marked in the BOM.
```

Author: Little too descriptive I feel. We do not need this line "While creating a Stock Entry for Transfer from Purchase Order, these items can be excluded one by one but it is impossible to do so if you have more than 100 items. "

Reviewer: better to be descriptive than leave doubts :)

> If there are some raw material which are sourced by the Supplier, such raw material has to be marked in the BOM.

Reviewer: ```suggestion
If some raw materials which are sourced by the Supplier, such raw materials have to be marked in the BOM.
```

> * It will have zero value in BOM

Reviewer: how will the cost of this item be factored into total manufacturing costs?
if supplier is providing the item, the manufacturer is still paying the costs for it right?

Author: @prasadarr It is for the sub-contract and from the user perspective. This kind of raw material is not in the user's stock but it is part of the BOM because it is being used in the manufacture. Suppliers can choose to put it in their Sales Order or it can be included in the price of the manufactured good. But for the user, it has zero value in BOM.
